### DISCUSSION TITLE
[ Shaders ] FMVs not using full range 0-255

### DISCUSSION THREAD

**Discussion by DLPB2** [julianxhokaxhiu/FFNx#192] [General]
http://forums.qhimm.com/index.php?topic=19970.msg279213#msg279213

**Comment by julianxhokaxhiu** [comment]
The shader is actually using transformation matrices and the full range for colors quite fine. See https://github.com/julianxhokaxhiu/FFNx/blob/master/misc/FFNx.frag#L41

**Comment by DLPB2** [comment]
Are you sure?  When I load Aali dll - brightness is correct - same as VLC player.  With FFNx...  it's not.

**Comment by julianxhokaxhiu** [comment]
Yes I am sure. Most probably your videos are not using JPEG color range. That's the only way to get full range color.
